frontend/test/acceptance/util/Browser.scala

```diff
@@ -34,13 +37,32 @@ trait Browser extends WebBrowser {
       throw new MissingPageElementException(q)
   }
 
-  def setValue(q: Query, value: String) {
-    if (pageHasElement(q))
+  def setValue(q: Query, value: String, clear: Boolean = false) {
+    if (pageHasElement(q)) {
+
+      if (clear) q.webElement.clear
       q.webElement.sendKeys(value)
-    else
+
+    } else
       throw new MissingPageElementException(q)
   }
 
+  /*
+    * Stripe wants you to pause between month and year and between each quartet in the cc
+    * This causes pain when you use Selenium. There are a few stack overflow posts- but nothing really useful.
+    * This pausing also seems to be necessary to make PayPal work properly,
+    * without this it starts to complain about invalid credentials after only
+    * a few characters.
+    * */
+  def setValueSlowly(q: Query, value: String): Unit = {
```

Comment:
Interesting that the issue exists for both PayPal and Stripe. @Ap0c could you ask PayPal support why is this necessary, while @jacobwinch does the same for Stripe.